### ISSUE TITLE
Customization of result/search navigation keys

### ISSUE THREAD

**Issue by Nomad-0** [adirh3/Fluent-Search#90]
**Is your feature request related to a problem? Please describe.**
I feel more comfortable using the `tab` key to navigate through results as it's closer to the home row than arrow keys. 

**Describe the solution you'd like**
As everyone has different preferences for keys, the most reasonable solution would be IMO to add an option to customize the following keys-

**navigate through result:** 

- up-?
- down-?
- right-?
- left-?
- insert tag-?
- show preview in search window-?

**Describe alternatives you've considered**

- up- `shift+tab`
- down- `tab`
- right- `right arrow`
- left- `left arrow`
- insert tag-  `space` / `ctrl+tab` / `ctrl+right arrow` / `shift+space` 
- show preview in search window- `shift+space`

**Additional context**

**Comment by Metroid-Prime** [comment]
Yes, an option would be better as I and others here use the arrow keys.

I like the arrow keys because down makes you go down the result and right arrow key allows you to interface with the command functions (like listary).
